py3status/modules/window_title.py

```diff
@@ -51,6 +51,9 @@ def window_title(self):
 
         transformed = False
         if window and 'name' in window and window['name'] != self.text:
```

Comment:
It is of no use at all, this was _used_ a long time ago for debugging only. You can drop it guys.